cmd/tick/main.go

```diff
@@ -4,10 +4,24 @@ package main
 
 import (
 	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
 	"time"
 )
 
 func main() {
+	c := make(chan os.Signal)
+	signal.Notify(c, os.Interrupt, syscall.SIGTERM)
+	go func() {
+		for sig := range c {
```

Comment:
I believe that will prevent the `tick` program from terminate on ^c in the terminal. This little command line tool is too useful to make it broken :)

If you were just doing this for testing one-off, I'd revert this change. If it's useful going forward, we should make another tool like "hang" or similar.

Reply:
Fair enough